src/libraries/System.Text.RegularExpressions/gen/Resources/xlf/Strings.cs.xlf

```diff
@@ -118,8 +118,8 @@
         <note />
       </trans-unit>
       <trans-unit id="InvalidLangVersionMessage">
-        <source>C# LangVersion of 10 or greater is required</source>
-        <target state="translated">Je požadována verze jazyku C# LangVersion 10 nebo vyšší.</target>
+        <source>C# LangVersion of 11 or greater is required</source>
+        <target state="needs-review-translation">Je požadována verze jazyku C# LangVersion 10 nebo vyšší.</target>
```

Comment:
should this just be `10` -> `11` for all languages? Or do we need  an official translation?

Reply:
I don't speak these languages and didn't want to assume.